README.md

```diff
@@ -0,0 +1,40 @@
+Jinja2 extension to handle git-specific things
+==============================================
+
+[![image](https://travis-ci.org/sobolevn/jinja2-git.svg?branch=master)](https://travis-ci.org/sobolevn/jinja2-git)
```

Comment:
Please, rename alts to match the domain: `travis` for travis, `coveralls` for coveralls, etc. Alts are `[image]` tags